### PR TITLE
multisig management tool milestone 2

### PR DESCRIPTION
**Pull request by carlhong**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/11

### CODE REVIEW

**Comment by mmagician** [comment]
Thanks for the second milestone, we'll check it as soon as we can!

File: deliveries/multisignature_management_tool_milestone_2.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** Please, provide a link to the merged contract (the `.md` file in the [applications](https://github.com/w3f/Grants-Program/blob/master/applications/multisignature_management_tool.md) directory).
```
Comment (alxs) [inline deliveries/multisignature_management_tool_milestone_2.md:5]:
```suggestion
* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/multisignature_management_tool.md
```

File: deliveries/multisignature_management_tool_milestone_2.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** Please, provide a link to the merged contract (the `.md` file in the [applications](https://github.com/w3f/Grants-Program/blob/master/applications/multisignature_management_tool.md) directory). 
+* **Milestone Number:** 2
+
+
+| Number | Deliverable | Link | Notes |
+| --- | --- | --- | --- |
+| 0. | Support more kinds of Extrinsic | [pr](https://github.com/itering/subscan-multisig-react/commit/147ce7e64e403b157d43389326d12a92da95fb64) |
```
Comment (alxs) [inline deliveries/multisignature_management_tool_milestone_2.md:11]: For what extrinsics has support been added with this PR exactly? The application mentions "Extrinsics such as staking, Democracy".

**Review by alxs — CHANGES_REQUESTED** [review]

File: deliveries/multisignature_management_tool_milestone_2.md (hunk 2 above)
Comment (carlhong) [reply deliveries/multisignature_management_tool_milestone_2.md]: We imported and modified react-components from polkadot apps to support all extrinsics exists in listed networks.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks again for the delivery. I started to look into it, [see](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/evaluations/multisignature_management_tool_2_Noc2.md). I so far have the following questions/comments:

- Which node version should I use for the installation (yarn)? I so far tried 14.9 .0 and 16.6.1
- The [tutorial](https://github.com/itering/subscan-multisig-react#Tutorial) seems to be a little bit outdated. Could you update it? For example "*Currently, only transfer operations are supported. We will support more types of extrinsic in the second stage."
- And just a general comment. It’s nice that you now support all extrinsic types, but I think now it’s quite difficult to use for the average user, since you have to know all the extrinsics and what they are doing.

**Comment by carlhong** [comment]
@Noc2  Thanks for the quick review and feedbacks, we've made corresponding updates. As for the node version, I am using yarn with node v14.16.0.

**Comment by Noc2** [comment]
Thanks for the quick response. Everything works with your node version. I’m happy to tell you the milestone is a pass and I will forward your invoice internally.  It still might make sense to further improve the usability of the UI.

**Comment by carlhong** [comment]
Glad to hear that. We will continue working on Subscan multisig project, and there are many [UI enhancement features](https://github.com/itering/subscan-multisig-react/issues) is in progress now.

**Comment by RouvenP** [comment]
hi @carlhong we just sent a test transaction of 0.001 BTC, could you confirm if received?

**Comment by carlhong** [comment]
@RouvenP Yes, we received 0.001 BTC.

**Comment by RouvenP** [comment]
@carlhong great - we sent the remainder. Apologies for the delay.

**Pull request merged**